WoWsShipBuilder.Core/DataContainers/Aircraft/AirstrikeDataContainer.cs

```diff
@@ -62,8 +63,8 @@ public bool IsExpanderOpen
         public static async Task<AirstrikeDataContainer?> FromShip(Ship ship, List<(string, float)> modifiers, bool isAsw, IAppDataService appDataService)
         {
             string header = isAsw ? "ShipStats_AswAirstrike" : "ShipStats_Airstrike";
-            Dictionary<string, AirStrike>? airstrikes = ship.AirStrikes;
-            if (ship.AirStrikes == null || ship.AirStrikes.Count == 0)
+            Dictionary<string, AirStrike> airstrikes = ship.AirStrikes;
+            if (ship.AirStrikes.Count == 0)
```

Comment:
Wouldn't this cause a nullPointer in case ship.AirStrikes is null? or is ship.AirStrikes never null? Going by the previous code, it could

Reply:
Cannot happen due to default values being assigned when there is no value set